wordpress-mobile/WordPress-iOS
# PPP: Set parent page analytics

This PR adds 3 new Analytics events to the Set Parent page view controller.
The new events are:
- _site_pages_set_parent_viewed_: When the VC appears
- _site_pages_set_parent_search_accessed_: When the set parent search is accessed
- _site_pages_set_parent_done_pressed_: When the done button is pressed

**To test:**
- Select a Blog -> Site pages -> click the ••• menu on a page and open the set parent view controller.
- You should be able to see the tracked events on your console
